mmrazor/models/ops/gather_tensors.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Any
+
+import torch
+import torch.distributed as dist
+
+
+class GatherTensors(torch.autograd.Function):
```

Comment:
UT

Reply:
There are no asserts or warnings in GatherTensor and its parent except NotImplementedError. 
Add its UT in test_distillation_loss.py to check whether the default process group has been initialized.